WordPressKit/JetpackBackupServiceRemote.swift

```diff
@@ -0,0 +1,91 @@
+import Foundation
+import WordPressShared
+import CocoaLumberjack
+
+open class JetpackBackupServiceRemote: ServiceRemoteWordPressComREST {
+    
+    /// Prepare a downloadable backup snapshot for a site.
+    ///
+    /// - Parameters:
+    ///     - siteID: The target site's ID.
+    ///     - rewindID: The rewindID of the snapshot to download.
+    ///     - types: The types of items to restore.
+    ///     - success: Closure to be executed on success.
+    ///     - failure: Closure to be executed on error.
+    ///
+    /// - Returns: A backup snapshot object.
+    ///
+    open func prepareBackup(_ siteID: Int,
+                            rewindID: Int? = nil,
+                            types: JetpackRestoreTypes? = nil,
+                            success: @escaping (_ backup: JetpackBackup) -> Void,
+                            failure: @escaping (Error) -> Void) {
+        let path = self.backupPath(for: siteID)
```

Comment:
We can remove the self. references since they're not needed

```suggestion
        let path = backupPath(for: siteID)
```